src/libraries/listener.library.ts

```diff
@@ -80,22 +152,36 @@ export class KeyboardShortcutListener implements IKeyboardShortcutListener {
         }
     }
 }
-
+/**
+ * @interface represents the handle of a listener
+ */
 export interface IListenerHandle {
+    /**
+     * @description removes this listener
+     */
     remove(): void;
 }
-
+/**
+ * @type { event: KeyboardEvent } it is handler to keep track of the listeners
```

Comment:
this description should be "the function that should be run when the keyboard combination is matched"